packages/owlbot-bootstrapper/cli/helper.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {CliArgs} from './run-trigger-command';
+import {execSync} from 'child_process';
+
+export class Helper {
+  args: CliArgs;
+  languageContainers: {
+    language: string;
+    languageContainer: string;
+    repoToClone: string;
+  }[];
+
+  constructor(args: CliArgs) {
+    this.args = args;
+    this.languageContainers = [
+      {
+        language: 'nodejs',
+        languageContainer: 'gcr.io/owlbot-bootstrap-prod/node-bootstrapper',
+        repoToClone: 'git@github.com/googleapis/google-cloud-node.git',
+      },
+    ];
+  }
+
+  public async getLanguageSpecificValues() {
+    for (const languageContainer of this.languageContainers) {
+      if (languageContainer.language === this.args['language']) {
+        languageContainer.languageContainer =
+          await this.getFullLatestContainerName(
+            languageContainer.languageContainer
+          );
+        return languageContainer;
+      }
+    }
+
+    // If there was never a match, and there was no language container specified in the CLI, we should throw
+    if (!this.args['languageContainer']) {
+      throw new Error('No language-specific container specified.');
+    }
+
+    return undefined;
+  }
+
+  public async getFullLatestContainerName(container: string) {
```

Comment:
this method does not need to be an instance method -- it doesn't require any of the object's properties. Consider making it a static method or a separate exported helper.

Reply:
I wrote it this way so that it was easier to test, or else you can't mock this function while testing. I did a fair bit of experimentation with the testing, so let me know if you see another way to test this additional functionality without making it a part of a helper class.

Reply:
Sinon can mock static methods (`sandbox.stub(Helper, 'getFullLatestContainerName').resolves(something)` and other exported functions (`import * as helperModule from 'helper'; sandbox.stub(helperModule, 'someExportedFunction').resolves(something);`)